examples/golang/components/http-password-checker/README.md

```diff
@@ -0,0 +1,91 @@
+# TinyGo HTTP Password Checker
+
+This repository contains a WebAssembly Component written in [TinyGo][tinygo], which:
+
+- Implements a [`wasi:http`][wasi-http]-compliant HTTP handler
+- Uses the [`httpserver` provider][httpserver-provider] to serve requests
+- Can be declaratively provisioned with [`wadm`][wadm]
+
+[wasi-http]: https://github.com/WebAssembly/wasi-http
+[httpserver-provider]: https://github.com/wasmCloud/wasmCloud/tree/main/crates/provider-http-server
+[blobstore-provider]: https://github.com/wasmCloud/wasmCloud/tree/main/crates/provider-blobstore-fs
+[wadm]: https://github.com/wasmCloud/wadm
+[tinygo]: https://tinygo.org/getting-started/install/
+[wash]:  https://wasmcloud.com/docs/ecosystem/wash/
+
+# Dependencies
+
+Before starting, ensure that you have the following installed:
+
+- The [TinyGo toolchain][tinygo]
+- [`wash`, the WebAssembly SHell][wash] installed.
```

Comment:
I presume this supposed to be rather
```suggestion
- [`wash`, the WAsmcloud SHell][wash] installed.
```

Which is a bit confusing since in the docs url it is 
"wasmCloud Shell" and in the CLI ascii art it is "wasmCloud Shell" 🤔

Reply:
Oh that is the place where it is called `WAsmcloud SHell` at the top. [docs](https://wasmcloud.com/docs/installation) I did remember seen another version of the wash abbreviation.

Reply:
Hey thanks for this @markkovari -- I put the change in!